must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -191,20 +191,38 @@ for ns in $namespaces; do
         done
     done
 
-    # Add Ready nodes to the list
+    # Add ocs ready nodes to the list
     nodes=$(oc get nodes -l cluster.ocs.openshift.io/openshift-storage='' --no-headers | awk '/\yReady\y/{print $1}')
 
+    for i in $(timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "ceph crash ls --connect-timeout=15"| awk '{print $1}'); do
+        { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "ceph crash info $i --connect-timeout=15" >> "${COMMAND_OUTPUT_DIR}"/crash_"${i}"; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1;
+    done
+
+    oc delete -f pod_helper.yaml
+
     # Collecting ceph prepare volume logs
     volume_collection(){
+        NODE_OUTPUT_DIR=${CEPH_COLLECTION_PATH}/volume_collection_${node}
+        mkdir -p "${NODE_OUTPUT_DIR}"
         printf "collecting prepare volume logs from node %s \n"  "${node}"
-        oc rsync -n "${ns}" "$(oc get pods -n "${ns}"| grep "${node//./}-debug"| awk '{print $1}')":/host/var/lib/rook/openshift-storage/log "${NODE_OUTPUT_DIR}"
+        { oc rsync -n "${ns}" "$(oc get pods -n "${ns}"| grep "${node//./}-debug"| awk '{print $1}')":/host/var/lib/rook/openshift-storage/log "${NODE_OUTPUT_DIR}"; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
     }
 
-    for i in $(timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "ceph crash ls --connect-timeout=15"| awk '{print $1}'); do
-        { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "ceph crash info $i --connect-timeout=15" >> "${COMMAND_OUTPUT_DIR}"/crash_"${i}"; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1;
-    done
+    # Collecting journal logs from the debug pods
+    journal_collection(){
+        JOURNAL_LOGS=${CEPH_COLLECTION_PATH}/journal_log_${node}
+        mkdir -p "${JOURNAL_LOGS}"
+        printf "collecting journal logs from %s \n" "${node}"
+        { oc rsync -n "${ns}" "$(oc get pods -n "${ns}"| grep "${node//./}-debug"| awk '{print $1}')":/host/var/log/journal.log "${JOURNAL_LOGS}"; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
```

Comment:
this file will be covered in this line 224 of your PR

Reply:
I have removed the `journal_collection` function and kept the `kernel_collection` only.